crates/bevy_text/src/pipeline.rs

```diff
@@ -141,6 +142,10 @@ impl TextPipeline {
 
             // Save spans that aren't zero-sized.
             if scale_factor <= 0.0 || text_font.font_size <= 0.0 {
+                once!(warn!(
+                    "Text span {entity} has a font size <= 0.0. Nothing will be displayed.",
```

Comment:
Maybe it might be more useful to quote the text from the text span than its entity id (or give both)?

Reply:
or make the size random each frame, that should help quickly noticing where's the error 😄